diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index 9b28df0e8aa4734af2c7006fd077ff17e2281db5..0000000000000000000000000000000000000000
diff --git a/esm/useIntersection.d.ts b/esm/useIntersection.d.ts
index 9d3ac04942d2f3118e5fb5bf3c1758e4513d1fa5..62d128868aa774d80674896a3ce532a6125526aa 100644
--- a/esm/useIntersection.d.ts
+++ b/esm/useIntersection.d.ts
@@ -1,3 +1,3 @@
 import { RefObject } from 'react';
-declare const useIntersection: (ref: RefObject<HTMLElement>, options: IntersectionObserverInit) => IntersectionObserverEntry | null;
+declare const useIntersection: (ref: RefObject<HTMLElement | null>, options: IntersectionObserverInit) => IntersectionObserverEntry | null;
 export default useIntersection;
diff --git a/lib/useIntersection.d.ts b/lib/useIntersection.d.ts
index 9d3ac04942d2f3118e5fb5bf3c1758e4513d1fa5..62d128868aa774d80674896a3ce532a6125526aa 100644
--- a/lib/useIntersection.d.ts
+++ b/lib/useIntersection.d.ts
@@ -1,3 +1,3 @@
 import { RefObject } from 'react';
-declare const useIntersection: (ref: RefObject<HTMLElement>, options: IntersectionObserverInit) => IntersectionObserverEntry | null;
+declare const useIntersection: (ref: RefObject<HTMLElement | null>, options: IntersectionObserverInit) => IntersectionObserverEntry | null;
 export default useIntersection;
